vue2-vuetify/src/dynaform/DateTimeControlRenderer.vue

```diff
@@ -43,64 +67,222 @@ import {
+      formatDateTime,
+      parseDateTime,
+      blurChangeFormat,
     };
   },
+  watch: {
+    date() {
+      this.dateFormatted = this.formatDate(this.date);
+      if (this.inputFormat == 'date-time') {
+        this.dateTimeFormatted = this.formatDateTime(
+          this.date + ' ' + this.time
+        );
+      }
+    },
+    time() {
+      this.timeFormatted = this.formatTime(this.time);
+      if (this.inputFormat == 'date-time') {
+        this.dateTimeFormatted = this.formatDateTime(
+          this.date + ' ' + this.time
+        );
+      }
+    },
+    dateTime() {
+      this.dateTimeFormatted = this.formatDateTime(this.dateTime);
+    },
+  },
   computed: {
     hint(): string {
       return this.control.uischema.options?.hint ?? '';
     },
     placeholder(): string {
       return this.control.uischema.options?.placeholder ?? '';
     },
-    dataTime: {
-      get(): string | null | undefined {
-        const datetimeLocalFormat = 'YYYY-MM-DDTHH:mm:ss.SSS';
-        const saveFormat = this.appliedOptions.dateTimeSaveFormat ?? undefined;
-        const value = this.control.data as string | undefined | null;
+    inputFormat(): string | undefined {
+      return this.control.schema?.format ?? '';
+    },
+    valueChangeFormat(): string | null {
+      if (this.inputFormat == 'date') {
+        return this.dateFormatted;
+      } else if (this.inputFormat == 'time') {
+        return this.timeFormatted;
+      } else if (this.inputFormat == 'date-time') {
+        return this.dateTimeFormatted;
+      } else {
+        return null;
+      }
+    },
+    maxDate(): string {
+      let max = this.control.uischema.options?.maxDate;
+      if (max) {
+        let min = this.control.uischema.options?.minDate;
+        let current = new Date(this.date);
+        max = new Date(max);
+
+        if (current > max) {
+          min ? (min = new Date(min)) : (min = '');
+          if (!min || !(max < min)) {
+            this.date = max.toISOString().split('T')[0];
+          }
+        }
+
+        if (min) {
+          max < min ? (max = min.toISOString()) : (max = max.toISOString());
+        }
+
+        return max ?? '';
+      } else {
+        return max ?? '';
+      }
+    },
+    minDate(): string {
+      let min = this.control.uischema.options?.minDate;
+      if (min) {
+        let max = this.control.uischema.options?.maxDate;
+        let current = new Date(this.date);
+        min = new Date(min);
 
-        const dateTime = parseDateTime(value, saveFormat);
-        return dateTime ? dateTime.local().format(datetimeLocalFormat) : value;
-      },
-      set(value: string) {
-        const datetimeLocalFormats = [
-          'YYYY-MM-DDTHH:mm:ss.SSS',
-          'YYYY-MM-DDTHH:mm:ss',
-          'YYYY-MM-DDTHH:mm',
-        ];
-        const saveFormat =
-          this.appliedOptions.dateTimeSaveFormat ?? 'YYYY-MM-DDTHH:mm:ssZ';
+        if (current < min) {
+          max ? (max = new Date(max)) : (max = '');
```

Comment:
change for like this
`max = max < min ? min.toISOString() : max.toISOString();`